syntax = "proto3";

package BeeGees.Commands.Responses;

message ShipmentCreatedResponse {
    string ShipmentId = 1;
    string CreatedDate = 2;
    bool Success = 3;
}